Makefile

```diff
@@ -95,7 +95,8 @@ build.version:
 	@mkdir -p $(OUTPUT_DIR)
 	@echo "$(VERSION)" > $(OUTPUT_DIR)/version
 
-build.common: build.version helm.build mod.check
+# build.common: build.version helm.build mod.check
```

Comment:
helm fails with `Error: Invalid Semantic Version` perhaps because we run on dirty checkout?